src/PrestaShopBundle/Service/Hook/Finder.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+namespace PrestaShopBundle\Service\Hook;
+
+/**
+ * This class declares the functions needed to get structured data
+ * from the modules, by asking them to follow a specific class.
+ */
+class Finder {
```

Comment:
Yep, I was wrong, finally this class can be used directly without being extended.

For the name, I had the same thoughts as you, but the namespace `Hook\Finder` was explicit enough in my opinion. Do you prefer to duplicate the `Hook` word is the class name ?